primitives/Cargo.toml

```diff
@@ -27,6 +27,9 @@ std = [
     "sp-core/std",
     "sp-runtime/std",
 ]
+try-runtime = [
+  "frame-support/try-runtime",
+]
```

Comment:
This crate (primitives) is not a pallet and it won't use the functionality that try-runtime provides. Consequently we don't need to compile `frame-support` with the `try-runtime` feature.

Reply:
So I removed `frame_support/try-runtime` from the feature `try-runtime`.